### ISSUE TITLE
MSID: 86687 Version: 1 DOI: 2023.02.20.529215

### ISSUE THREAD

**Issue by nlisgo** [elifesciences/enhanced-preprints-import#296]
"msas": "Evolutionary Biology", "Genetics and Genomics"
"msid": "86687"
"version": "1"
"preprintDoi": "10.1101/2023.02.20.529215"
"articleType": "Reviewed Preprint"
"status": "Published from the original preprint after peer review and assessment by eLife."

"Reviewed Preprint posted": "2023-05-30"
"Sent for peer review": "2023-02-20"
"Posted to bioRxiv": "2023-02-21" (link: "Go to bioRxiv": "https://www.biorxiv.org/content/10.1101/2023.02.20.529215v1")

[PLACE PDF URL HERE WHEN AVAILABLE] See step 7

## Step 1. Inform bioRxiv

Who can help: @QueenKraken, @nlisgo, @scottaubrey

- [x] Manuscript is in data hub index (https://data-hub-api.elifesciences.org/enhanced-preprints/docmaps/v1/by-publisher/elife/get-by-manuscript-id?manuscript_id=86687)

## Step 2. Create preview of manuscript

Who can help: @fred-atherden, @nlisgo, @scottaubrey

- [ ] Manuscript is available for preview (https://prod--epp.elifesciences.org/preview/86687/v1)
    - [ ] Trigger import for staging: https://prod--epp.elifesciences.org/import
    - [ ] Sync data folder with s3://prod-elife-epp-data/data
    - [ ] Trigger import for staging: https://staging--epp.elifesciences.org/import
    - [ ] Sync data folder with s3://staging-elife-epp-data/data
    - [ ] Pull request created to [enhanced-preprint-data](https://github.com/elifesciences/enhanced-preprints-data/pulls)
    - [ ] Post link to pull request below

Pull request: https://github.com/elifesciences/enhanced-preprints-data/pull/90

Detailed instructions: https://github.com/elifesciences/enhanced-preprints-data#add-a-manuscript

## Step 3: Awaiting public reviews

Who can help: Editorial team

- [x] Public reviews available on sciety (https://sciety.org/articles/activity/10.1101/2023.02.20.529215)
- [x] Manuscript and editor details at the top of this issue (Supplied by Editorial team)
- [ ] Reviews returned by EPP - https://prod--epp.elifesciences.org/api/reviewed-preprints/86687/v1/reviews

<details>
<summary>Example</summary>

```
"msas": "Genetics and Genomics", "Neuroscience"
"msid": "84628"
"version": "1"
"preprintDoi": "10.1101/2022.10.28.514241"
"articleType": "Reviewed Preprint"
"status": "Published from the original preprint after peer review and assessment by eLife."

"Reviewed Preprint posted": "2023-01-02"
"Sent for peer review": "2022-10-28"
"Posted to bioRxiv": "2022-11-21" (link: "Go to bioRxiv": "https://www.biorxiv.org/content/10.1101/2022.10.28.514241v1")

Editors:
Reviewing Editor
Michael B Eisen
University of California, Berkeley, United States
Senior Editor
Michael B Eisen
University of California, Berkeley, United States
```

</details>

## Step 4: Modify manuscripts.json (no PDF)

- [ ] Reviewed preprint is published to EPP (https://elifesciences.org/reviewed-preprints/86687)
    - [ ] Pull request created to [enhanced-preprint-client](https://github.com/elifesciences/enhanced-preprints-client/pulls)
    - [ ] Pull request switched to "Ready for review"
    - [ ] Post link to pull request below
- [ ] Request that a doi be registered
    - [ ] Post link to [#enhanced-preprint](https://elifesciences.slack.com/archives/C03EVJSUA77) comment below

Pull request: [PLACE LINK TO PULL REQUEST HERE]
[#enhanced-preprint](https://elifesciences.slack.com/archives/C03EVJSUA77) comment thread: [PLACE LINK TO COMMENT HERE]

<details>
<summary>Instructions to modify manuscripts.json</summary>

- Visit: https://github.com/elifesciences/enhanced-preprints-client/actions/workflows/publish-manuscript.yaml
- Click: Run workflow
- Complete the form and click "Run workflow"
- A successful run should result in a new pull request at https://github.com/elifesciences/enhanced-preprints-client/pulls
- Open the pull request and click the "Ready for review" button to trigger tests
- Once the tests pass and you are happy with the changes the PR can be merged

Example pull request: https://github.com/elifesciences/enhanced-preprints-client/pull/334/files

Once the pull request is merged in it should be available a few minutes later.

</details>

### Request that a doi

Post the following in [#enhanced-preprint](https://elifesciences.slack.com/archives/C03EVJSUA77):

```
@Fred can you register a doi for https://elifesciences.org/reviewed-preprints/86687
```

## Step 5: Awaiting search reindex

- [ ] Reviewed preprint is avaliable on journal homepage (https://elifesciences.org)

The search reindex is triggered once an hour. We need the reviewed preprint to be indexed as the search application serves the journal homepage.

<details>
<summary>Additional info</summary>

If needed, the jenkins pipeline to reindex search can be triggered sooner.

https://alfred.elifesciences.org/job/process/job/process-reindex-reviewed-preprints/

</details>

## Step 6: Published! Request PDF generation

- [ ] Post the link to the PDF url at the top of the issue
    - [ ] Post link to [#enhanced-preprint](https://elifesciences.slack.com/archives/C03EVJSUA77) comment below

[#enhanced-preprint](https://elifesciences.slack.com/archives/C03EVJSUA77) comment thread: [PLACE LINK TO COMMENT HERE]

Post the following to the [#enhanced-preprint](https://elifesciences.slack.com/archives/C03EVJSUA77) on slack:

```
@Ryan Dix-Peek please can you generate a PDF for https://elifesciences.org/reviewed-preprints/86687
```

## Step 7: Introduce PDF to data folder and git repo

- [ ] PDF is avaliable at https://github.com/elifesciences/enhanced-preprints-data/raw/master/data/86687/v1/86687-v1.pdf

Detailed instructions: https://github.com/elifesciences/enhanced-preprints-data#add-a-pdf

## Step 8: Add PDF url to manuscripts.json

- [ ] Reviewed preprint PDF is available for download (https://prod--epp.elifesciences.org/reviewed-preprints/86687)
    - [ ] Pull request created to [enhanced-preprint-client](https://github.com/elifesciences/enhanced-preprints-client/pulls)
    - [ ] Pull request switched to "Ready for review"
    - [ ] Post link to pull request below

[PLACE LINK TO PULL REQUEST HERE]

<details>
<summary>Instructions to add PDF url to manuscripts.json</summary>

- Visit: https://github.com/elifesciences/enhanced-preprints-client/actions/workflows/add-pdf-url-to-manuscript.yaml
- Click: Run workflow
- Complete the form and click "Run workflow"
- A successful run should result in a new pull request at https://github.com/elifesciences/enhanced-preprints-client/pulls
- Open the pull request and click the "Ready for review" button to trigger tests
- Once the tests pass and you are happy with the changes the PR can be merged

Example pull request: https://github.com/elifesciences/enhanced-preprints-client/pull/397/files

Once the pull request is merged in it should be available a few minutes later.

</details>

## Step 9: Done!

- [ ] Kettle is on!

**Comment by fred-atherden** [comment]
Hi Andy,

I'm assuming this because the response is not present in the docmap (for some reason):
https://data-hub-api.elifesciences.org/enhanced-preprints/docmaps/v1/by-publisher/elife/get-by-manuscript-id?manuscript_id=86687

<details>

<summary>JSON</summary>

```json
"_:b2": {
      "actions": [
        {
          "participants": [
            {
              "actor": { "name": "anonymous", "type": "person" },
              "role": "peer-reviewer"
            }
          ],
          "outputs": [
            {
              "type": "review-article",
              "published": "2023-05-26 08:47:32.174911+00:00",
              "doi": "10.7554/eLife.86687.1.sa1",
              "license": "http://creativecommons.org/licenses/by/4.0/",
              "url": "https://doi.org/10.7554/eLife.86687.1.sa1",
              "content": [
                {
                  "type": "web-page",
                  "url": "https://hypothes.is/a/9BmMEPuhEe2-8pNsuN5NiQ"
                },
                {
                  "type": "web-page",
                  "url": "https://sciety.org/articles/activity/10.1101/2023.02.20.529215#hypothesis:9BmMEPuhEe2-8pNsuN5NiQ"
                },
                {
                  "type": "web-page",
                  "url": "https://sciety.org/evaluations/hypothesis:9BmMEPuhEe2-8pNsuN5NiQ/content"
                }
              ]
            }
          ]
        },
        {
          "participants": [
            {
              "actor": { "name": "anonymous", "type": "person" },
              "role": "peer-reviewer"
            }
          ],
          "outputs": [
            {
              "type": "review-article",
              "published": "2023-05-26 08:47:32.940771+00:00",
              "doi": "10.7554/eLife.86687.1.sa2",
              "license": "http://creativecommons.org/licenses/by/4.0/",
              "url": "https://doi.org/10.7554/eLife.86687.1.sa2",
              "content": [
                {
                  "type": "web-page",
                  "url": "https://hypothes.is/a/9I5F5vuhEe2YEH87ouOu7w"
                },
                {
                  "type": "web-page",
                  "url": "https://sciety.org/articles/activity/10.1101/2023.02.20.529215#hypothesis:9I5F5vuhEe2YEH87ouOu7w"
                },
                {
                  "type": "web-page",
                  "url": "https://sciety.org/evaluations/hypothesis:9I5F5vuhEe2YEH87ouOu7w/content"
                }
              ]
            }
          ]
        },
        {
          "participants": [
            {
              "actor": {
                "name": "Christian Landry",
                "type": "person",
                "_relatesToOrganization": "Université Laval, Canada"
              },
              "role": "editor"
            },
            {
              "actor": {
                "name": "Christian Landry",
                "type": "person",
                "_relatesToOrganization": "Université Laval, Canada"
              },
              "role": "senior-editor"
            }
          ],
          "outputs": [
            {
              "type": "evaluation-summary",
              "published": "2023-05-26 08:47:33.933968+00:00",
              "doi": "10.7554/eLife.86687.1.sa3",
              "license": "http://creativecommons.org/licenses/by/4.0/",
              "url": "https://doi.org/10.7554/eLife.86687.1.sa3",
              "content": [
                {
                  "type": "web-page",
                  "url": "https://hypothes.is/a/9SXyLvuhEe2PzKcqK2YCdQ"
                },
                {
                  "type": "web-page",
                  "url": "https://sciety.org/articles/activity/10.1101/2023.02.20.529215#hypothesis:9SXyLvuhEe2PzKcqK2YCdQ"
                },
                {
                  "type": "web-page",
                  "url": "https://sciety.org/evaluations/hypothesis:9SXyLvuhEe2PzKcqK2YCdQ/content"
                }
              ]
            }
          ]
        }
      ],
      "assertions": [
        {
          "item": {
            "type": "preprint",
            "doi": "10.1101/2023.02.20.529215",
            "versionIdentifier": "1"
          },
          "status": "peer-reviewed"
        }
      ],
      "inputs": [
        {
          "type": "preprint",
          "doi": "10.1101/2023.02.20.529215",
          "url": "https://www.biorxiv.org/content/10.1101/2023.02.20.529215v1",
          "versionIdentifier": "1"
        }
      ],
      "previous-step": "_:b1"
    }
```

</details>

Maybe this is something for @hazalCiplak can help with?

**Comment by HazalCiplak** [comment]
Hi @fred-atherden and @acollings ,

Thank you for bringing this to our attention.
The reason seems like the `tags` hasn’t been defined in hypothesis. Please find the related screenshot below. In the DocMaps API we are using tags for differentiate summary, review or author response. If a tag is null we are not able to import it into DocMaps result. Once the tags are updated, this information should properly appear in the results. 

Could you please let me know who we should contact to address this update? 

<img width="908" alt="Screenshot 2023-05-31 at 12 40 11" src="https://github.com/elifesciences/enhanced-preprints-import/assets/6259479/55d208d1-44b5-4d8f-ae85-451b14252e87">

**Comment by fred-atherden** [comment]
Thanks Hazal! 

I'm not sure. @acollings do you know? Is this something to check with Paul about? Is it an issue with Kotahi?

**Comment by acollings** [comment]
Thanks @fred-atherden and @HazalCiplak: that would have been human error (Sue's away this week so someone else is posting author responses). Now I've fixed it, should this pull in automatically?

**Comment by fred-atherden** [comment]
Thanks Andy. 

@HazalCiplak can confirm, but I'd expect dataHub to update later today, and then the RP to be updated next subsequent time data is loaded/the app is deployed.

**Comment by HazalCiplak** [comment]
Thank you. I can confirm tags has been updated in Data Hub, so in DocMaps API result it should be updated soon. Please note that there is a small delay between DocMaps API result and Data Hub because of cashing the query.

<img width="854" alt="Screenshot 2023-05-31 at 15 41 05" src="https://github.com/elifesciences/enhanced-preprints-import/assets/6259479/f4d0b390-5e52-41dd-b5dd-e740bfc33435">
